crowdin.yaml

```diff
@@ -13,12 +13,5 @@ files:
       [
         /docs/.templates,
         /docs/api,
-        /docs/guides/architecture/recipes,
-        /docs/guides/development,
-        /docs/guides/debugging,
-        /docs/guides/testing,
-        /docs/guides/building,
-        /docs/guides/distribution,
-        /docs/guides/features,
-        /docs/guides/faq.md,
+        /docs/guides/distribution/updater,
```

Comment:
My bad because I said it incorrectly in the Discord message, but this should be `updater.md` since it's a file